$NetBSD$

Use GTK3, since that's how we build wxWidgets.

Define additional macro to OpenGL ES builds.

--- CMakeLists.txt.orig	2024-06-14 21:54:48.000000000 +0000
+++ CMakeLists.txt
@@ -53,7 +53,7 @@ CMAKE_DEPENDENT_OPTION(SLIC3R_DESKTOP_IN
 
 set(OPENVDB_FIND_MODULE_PATH "" CACHE PATH "Path to OpenVDB installation's find modules.")
 
-set(SLIC3R_GTK "2" CACHE STRING "GTK version to use with wxWidgets on Linux")
+set(SLIC3R_GTK "3" CACHE STRING "GTK version to use with wxWidgets on Linux")
 
 set(IS_CROSS_COMPILE FALSE)
 
@@ -112,6 +112,7 @@ endif ()
 
 if (SLIC3R_OPENGL_ES)
     add_definitions(-DSLIC3R_OPENGL_ES)
+    add_definitions(-DENABLE_OPENGL_ES)
 endif()
 
 if(SLIC3R_DESKTOP_INTEGRATION)
